src/components/document/SubTabs/components/Text/Text.tsx

```diff
@@ -6,15 +6,11 @@ import { useStyles } from './Text.styles'
 export const Text: FC<{ content: string }> = ({ content }) => {
     const { classes } = useStyles()
 
-    const escapeHtml = (unsafe: string) => {
```

Comment:
I have removed this since it seems to be unnecessary, and it was breaking the text search within the document preview. We need html markups in order to generate the highlights within the text viewer.

Also, this code was added in a previous commit in order to fix the results table row, but this component is not used in that ResultsTableRow